### PR TITLE
Batch backports to main-7.0.x 

### PR DESCRIPTION
**Pull request by jlucovsky**
Continuation of #12006 

Backports to main-7.0..x of 6509 and 7275

Link to tickets: 
- https://redmine.openinfosecfoundation.org/issues/6509
- https://redmine.openinfosecfoundation.org/issues/7275

Describe changes:
- Cherry-picks of commits for these issues

Updates:
- Fixup of cherry-pick that introduced dup symbols.

### Provide values to any of the below to override the defaults.

- To use an LibHTP, Suricata-Verify or Suricata-Update pull request,
  link to the pull request in the respective `_BRANCH` variable.
- Leave unused overrides blank or remove.

SV_REPO=
SV_BRANCH=
SU_REPO=
SU_BRANCH=
LIBHTP_REPO=
LIBHTP_BRANCH=

### CODE REVIEW

File: src/log-stats.c [hunk 1]
```diff
@@ -89,17 +89,16 @@ static int LogStatsLogger(ThreadVars *tv, void *thread_data, const StatsTable *s
     int days = in_hours / 24;
 
     MemBufferWriteString(aft->buffer, "----------------------------------------------"
-            "--------------------------------------\n");
+                                      "-----------------------------------------------------\n");
```
Comment (victorjulien) [inline src/log-stats.c:92]: this changes the format, which may break tools that might parse them. Should we backport this?

**Comment by jufajardini** [comment]
If I recall correctly, we are still pondering on whether the format that we have in master for the exception policy stats - especially for app-layer error - is what we want -- that's why the backport has been sitting for so long. We should resurface this discussion...

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 23160

File: src/log-stats.c (hunk 1 above)
Comment (jlucovsky) [reply src/log-stats.c:92]: I'm going to move the PR to draft mode as there is consideration for this and other related items.

**Comment by catenacyber** [comment]
Should this be rebased ? There was the merge of https://redmine.openinfosecfoundation.org/issues/7275 right ?

**Comment by victorjulien** [comment]
Needs to be reconsidered later.

**Pull request closed without merging**